### PR TITLE
feat(badge): implement typography

### PR DESCRIPTION
**Pull request by SisIvanova**
Closes #  

### Additional information (check all that apply):
 - [ ] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/core/styles/components/input/_input-group-theme.scss [hunk 1]
```diff
@@ -1454,8 +1454,10 @@
         padding-inline: pad-inline(rem(2px), rem(4px), rem(6px));
     }
 
-    %form-group-input--search {
-        padding-inline: pad-inline(rem(6px), rem(8px), rem(10px));
+    @if $variant == 'indigo-design' {
+        %form-group-input--search {
+            padding-inline: pad-inline(rem(6px), rem(8px), rem(10px));
+        }
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/components/input/_input-group-theme.scss:1460]: Is this intentional in the PR or is it a left over from another branch? I see that this PR specifically introduces changes to the badge typography.

**Review by simeonoff — COMMENTED** [review]
See comments

File: projects/igniteui-angular/src/lib/core/styles/components/input/_input-group-theme.scss (hunk 1 above)
Comment (SisIvanova) [reply projects/igniteui-angular/src/lib/core/styles/components/input/_input-group-theme.scss:1460]: Yes, it is intentional. It fixes a bug introduced with the Indigo design changes PR (#13948). If needed I will make a separate PR for this.

Comment (simeonoff) [reply projects/igniteui-angular/src/lib/core/styles/components/input/_input-group-theme.scss:1460]: Sneaky, but ok.

**Review by simeonoff — APPROVED** [review]

**Pull request merged**